sensu/plugins/check-ceph.rb

```diff
@@ -27,10 +27,18 @@
 #   normally considered Ceph warnings to be overlooked and considered
```

Comment:
osd near full test output
# ./check-ceph.rb -e "osds are down,near full osd,clock skew,mons down" -d -o -f -p
CheckCephHealth CRITICAL: HEALTH_WARN 1 near full osd(s)
osd.0 is near full at 71%
ID WEIGHT  REWEIGHT SIZE   USE    AVAIL  %USE  VAR
 0 0.85999  1.00000   883G   630G   253G 71.32 10.01
 1 0.85999  1.00000   883G 30822M   853G  3.41  0.48
 2 0.85999  1.00000   883G 29099M   855G  3.22  0.45
 3 0.85999  1.00000   883G 29762M   854G  3.29  0.46
 4 0.85999  1.00000   883G 29458M   854G  3.26  0.46
 5 0.85999  1.00000   883G 32049M   852G  3.54  0.50
 6 0.85999  1.00000   883G 30974M   853G  3.42  0.48
 7 0.85999  1.00000   883G 29613M   854G  3.27  0.46
 8 0.85999  1.00000   883G 30234M   854G  3.34  0.47
 9 0.85999  1.00000   883G 29894M   854G  3.30  0.46
10 0.85999  1.00000   883G 30625M   853G  3.39  0.48
11 0.85999  1.00000   883G 30684M   853G  3.39  0.48
12 0.85999  1.00000   883G 30739M   853G  3.40  0.48
13 0.85999  1.00000   883G 31677M   852G  3.50  0.49
14 0.85999  1.00000   883G 28778M   855G  3.18  0.45
15 0.85999  1.00000   883G 28758M   855G  3.18  0.45
16 0.85999  1.00000   883G 31808M   852G  3.52  0.49
17 0.85999  1.00000   883G 30267M   853G  3.35  0.47
              TOTAL 15903G  1133G 14770G  7.13
MIN/MAX VAR: 0.45/10.01  STDDEV: 15.57